x-pack/plugins/security_solution/server/lib/detection_engine/rule_types/indicator_match/threat_mapping/types.ts

```diff
@@ -119,7 +119,7 @@ export interface CreateThreatSignalOptions {
   threatPitId: OpenPointInTimeResponse['id'];
   reassignThreatPitId: (newPitId: OpenPointInTimeResponse['id'] | undefined) => void;
   allowedFieldsForTermsQuery: AllowedFieldsForTermsQuery;
-  inputIndexFields: DataViewFieldBase[];
+  // todo
   threatIndexFields: DataViewFieldBase[];
```

Comment:
Where do these come from? Are they worth investigating?

Reply:
edit: after thinking more my suggestion to focus on Query rules and ignore indicator match for now would supercede my comment above